shell/platform/darwin/ios/framework/Source/FlutterPlatformPlugin.h

```diff
@@ -12,7 +12,7 @@
 @interface FlutterPlatformPlugin : NSObject
 - (instancetype)init NS_UNAVAILABLE;
 + (instancetype)new NS_UNAVAILABLE;
-- (instancetype)initWithEngine:(fml::WeakNSObject<FlutterEngine>)engine NS_DESIGNATED_INITIALIZER;
+- (instancetype)initWithEngine:(FlutterEngine*)engine NS_DESIGNATED_INITIALIZER;
```

Comment:
We can make this (and probably some others) `(__weak FlutterEngine*)` once FlutterViewController.mm (which also #imports this) is migrated to ARC.